worker/src/polkadex_db/orderbook.rs

```diff
@@ -237,4 +254,23 @@ mod tests {
             vec![first_order(), second_order()]
         );
     }
+
+    #[test]
+    fn prepare_for_sending() {
+        let first_order = first_order();
+        let mut orderbook_mirror = OrderbookMirror {
+            general_db: GeneralDB::new(HashMap::new(), PermanentStorageMock::default()),
+        };
+        orderbook_mirror
+            .general_db
+            .db
+            .insert("FIRST_ORDER".encode(), first_order.encode());
+
+        assert_eq!(
+            orderbook_mirror.prepare_for_sending().unwrap(),
+            vec![OrderbookData {
```

Comment:
Wrap right side in `Ok()`/`Some()` instead of unwrap on left. so the assert will fail instead of the unwrap causing a panic

Reply:
Changed.